server/interaction/npc/npc.go

```diff
@@ -0,0 +1,100 @@
+package npc
+
+import (
+	"errors"
+
+	"github.com/joshprzybyszewski/cribbage/logic/scorer"
+	"github.com/joshprzybyszewski/cribbage/model"
+	"github.com/joshprzybyszewski/cribbage/server/interaction"
+	"github.com/joshprzybyszewski/cribbage/utils/rand"
+)
+
+var npcs = map[model.PlayerID]npcLogic{
+	`dumbNPC`:       &dumbNPCLogic{},
```

Comment:
could we define some consts for the dumb, simple, and calculated npc `PlayerID`s and use them here?

Probably best to define them in this package